code/modules/paperwork/photocopier.dm

```diff
@@ -11,9 +13,15 @@
 	var/obj/item/paper/copy = null //what's in the copier!
 	var/obj/item/photo/photocopy = null
 	var/obj/item/paper_bundle/bundle = null
-	var/copies = 1 //how many copies to print!
-	var/toner = 30 //how much toner is left! woooooo~
-	var/maxcopies = 10 //how many copies can be copied at once- idea shamelessly stolen from bs12's copier!
+	///how many copies to print!
+	var/copies = 1
+	//how much toner is left! woooooo~
```

Comment:
```suggestion
	///how much toner is left! woooooo~
```
Again - we are trying to use autodoc comments here. Again I encourage you to read https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/AUTODOC.md